keps/sig-contributor-experience/0000-community-forum/kep.yaml

```diff
@@ -0,0 +1,16 @@
+title: A community forum for Kubernetes
+authors:
```

Comment:
Can you please also fill-in kep-number: 0000 ?

Reply:
done 👍